webapp/channels/src/components/integrations/installed_incoming_webhook.test.tsx

```diff
@@ -29,7 +29,7 @@ describe('components/integrations/InstalledIncomingWebhook', () => {
     const teamId = 'testteamid';
 
     test('should match snapshot', () => {
-        function emptyFunction() {} //eslint-disable-line no-empty-function
+        function emptyFunction() { } //eslint-disable-line no-empty-function
```

Comment:
why the empty space here? if it is for being compliant with the linter, then the comment should be removed.